src/main/webapp/app/exercises/text/participate/text-editor.component.ts

```diff
@@ -173,6 +175,14 @@ export class TextEditorComponent implements OnInit, OnDestroy, ComponentCanDeact
         return null;
     }
 
+    get wordCount() {
+        return this.stringCountService.countWords(this.answer);
+    }
+
+    get characterCount() {
```

Comment:
```suggestion
    get characterCount(): number {
```

Reply:
Made it explicit as suggested in 47d184757fa9a405309179c532dd2da6c45f4a90